src/lio/lio_core_os.c

```diff
@@ -579,12 +580,118 @@ gop_op_generic_t *gop_lio_abort_regex_object_set_multiple_attrs(lio_config_t *lc
 }
 
 //*************************************************************************
-// lio_move_op - Renames an object
+//  lio_is_dir_empty - Returns 0 if the directory is empty and a non-0 otherwise
```

Comment:
This is (soon) to be part of the rigid API. Are you sure you don't want to move the names again?